### PR TITLE
REFPLTB-1474: Task to stabilize opensync 2.4.5 on Turris omnia and upstream it to meta-turris

### PR DESCRIPTION
**Pull request by rnaren94**
REFPLTB-1474: stabilize opensync 2.4.5 on Turris omnia

### CODE REVIEW

File: recipes-connectivity/opensync/opensync_2.4.5.bbappend [hunk 1]
```diff
@@ -0,0 +1,44 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/opensync_2.4.5:${THISDIR}/files:"
+#FILESEXTRAPATHS_prepend := "${THISDIR}/opensync_2.4.5:${THISDIR}/files:"
```
Comment (manigandanj) [inline recipes-connectivity/opensync/opensync_2.4.5.bbappend:2]: please remove this line.

**Review by manigandanj — CHANGES_REQUESTED** [review]
Don't remove this opensync_2.0.5.bbappend until meta-rdk-opensync in manifest XML is updated to point to osync_2.4.5

**Comment by scthunderbolt** [comment]
Please have a meaningful subject line in the commit-msg.

File: recipes-connectivity/opensync/opensync_2.4.5.bbappend (hunk 1 above)
Comment (manigandanj) [inline recipes-connectivity/opensync/opensync_2.4.5.bbappend]: please remove this line.

**Review by manigandanj — CHANGES_REQUESTED** [review]

**Comment by scthunderbolt** [comment]
Closed due to inactivity, please restore if still needed.

**Pull request closed without merging**